src/panel/pages/explorer/Explorer.tsx

```diff
@@ -1,38 +1,25 @@
-import React, { useState, useContext } from "react";
+import React, { useContext } from "react";
 import styled from "styled-components";
 import { Background } from "../../components";
 import { ExplorerContext } from "../../context";
-import { FieldNode } from "../../context/Explorer/ast";
-import { Tree } from "./Tree";
-import { DetailViewPane } from "./DetailView";
+import { Tree, NodeInfoPane } from "./components";
 
 export function Explorer() {
-  const operations = useContext(ExplorerContext);
-  const [focusedNodeId, setFocusedNodeId] = useState<string | undefined>(
```

Comment:
I coudn't work out how this is any different to detailViewNode and wanted to trim down state + renders - from what I can tell, both values get set at the same time